src/program/program.py

```diff
@@ -71,12 +73,12 @@ def initialize_services(self):
             Symlinker: Symlinker(),
             Updater: Updater(),
             Downloader: Downloader(),
-            # Depends on Symlinker having created the file structure so needs
-            # to run after it
-            SymlinkLibrary: SymlinkLibrary(),
+            SymlinkLibrary: SymlinkLibrary(), # Depends on Symlinker
             PostProcessing: PostProcessing(),
         }
 
+        self.services[RealDebridLibrary] = RealDebridLibrary(self.services[Downloader].services[RealDebridDownloader], self.services[TraktIndexer].run)
```

Comment:
With the Library, it needs the `match_existing_content` to be enabled for the app to run. Don't think that was intentional, as you wanted that to be optional right?

![image](https://github.com/user-attachments/assets/a5e32490-8484-471a-972b-71f19f524a00)